packages/web-react/src/components/Tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -1,42 +1,18 @@
 import '@testing-library/jest-dom';
 import React from 'react';
 import { render, fireEvent } from '@testing-library/react';
-import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+// import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
 import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
 import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
 import { Button } from '../../Button';
-import { Tooltip, TooltipModern, TooltipTrigger, TooltipPopover } from '..';
+import { Tooltip, TooltipTrigger, TooltipPopover } from '..';
 
 describe('Tooltip', () => {
-  classNamePrefixProviderTest(Tooltip, 'Tooltip');
+  stylePropsTest((props) => <Tooltip {...props} data-testid="Tooltip-test" />, 'Tooltip-test');
```

Comment:
should there be an ID, too? If it is required.

Reply:
id added 👍